src/main/java/com/parasoft/demoapp/config/rabbitmq/RabbitMQConfig.java

```diff
@@ -12,6 +13,7 @@
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
+@Getter
```

Comment:
Can we modify "connection()" method at line 39 to save "connection" as a variable and check whether existing connection is open every time? Instead of creating new connection every time, only create new connection if saved connection is not open anymore.